MultiSource/Benchmarks/Ptrdist/ft/ft.c

```diff
@@ -67,7 +67,7 @@ _Ptr<Vertices>  MST(_Ptr<Vertices>  graph);
 int debug = 1;
 
 int
-main(int argc, _Array_ptr<_Ptr<char>> argv )
+main(int argc, _Array_ptr<const char*> argv : count(argc) )
```

Comment:
This is good. If you can, you should `#include "stdchecked.h"` so that you can use `ptr<T>` instead of `_Ptr<t>`. Is there a problem with names overlapping?